action.yml

```diff
@@ -0,0 +1,133 @@
+name: 'Publish to NeoFS Github Action'
+description: 'This action allows you publish files, tests reports, static web pages and etc. to NeoFS'
+author: 'Oleg Kulachenko'
```

Comment:
Likely NeoSPCC would be better here, for me this implies some maintainer role as well (at least if there is no other field for it) and the role can be edited by many people in the future.

Reply:
> Likely NeoSPCC would be better here, for me this implies some maintainer role as well (at least if there is no other field for it) and the role can be edited by many people in the future.

Fixed